1_fetch/fetch_config.yaml

```diff
@@ -0,0 +1,68 @@
+sb_fetch.py:
```

Comment:
I would consider restructuring this config file with another level, the category. (`metadata`, `obs`,  `drivers`). 

See example here: https://github.com/jsadler2/lake-temperature-lstm-static/blob/snakefile-shorter/1_fetch/fetch_config.yaml

Reply:
This allows you to access the `sb_id` more easily: https://github.com/jsadler2/lake-temperature-lstm-static/blob/ef32613bf46a67df06a80195472ac4518718be28/Snakefile#L24

Reply:
Done! This structure does make things easier.